### PR TITLE
chore: remove all what was deprecated

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/api-bindings/codegen-api.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ generates:
         TxId: string
         UnixTimestamp: number
         URI: string
-        URL: Url
+        URL: string
```
Comment (cesarenaldi) [inline packages/api-bindings/codegen-api.yml:59]: Should `URI` and `URL` now be mapped to the `URI` and `URL` branded types in shared-kernel?

Comment (krzysu) [reply packages/api-bindings/codegen-api.yml:59]: right, will do!

Comment (krzysu) [reply packages/api-bindings/codegen-api.yml]: done

File: packages/react/src/index.ts [hunk 2]
```diff
@@ -57,7 +57,6 @@ export type {
   PromiseResult,
   Result,
   Success,
-  Url,
```
Comment (cesarenaldi) [inline packages/react/src/index.ts:60]: Verify if export `URI` and `URL` is needed here or already done elsewhere.

File: packages/react/src/transactions/adapters/schemas/common.ts [hunk 3]
```diff
@@ -13,22 +13,20 @@ import {
...
 
 export const EvmAddressSchema: z.ZodType<EvmAddress, z.ZodTypeDef, string> = z
   .string()
   .min(42)
   .transform((value) => value);
 
-export const DataSchema: z.ZodType<Data, z.ZodTypeDef, string> = z
-  .string()
-  .transform((value) => value as Data);
+export const DataSchema: z.ZodType<Data, z.ZodTypeDef, string> = z.string().transform(data);
```
Comment (cesarenaldi) [inline packages/react/src/transactions/adapters/schemas/common.ts:29]: 👍

**Review by cesarenaldi — APPROVED** [review]

File: packages/react/src/index.ts (hunk 2 above)
Comment (krzysu) [reply packages/react/src/index.ts:60]: they are now only used internally in our packages, don't need to export them at all

**Pull request merged**